docs/en/modules/mdns.md

```diff
@@ -9,14 +9,17 @@
 Register a hostname and start the mDNS service. If the service is already running, then it will be restarted with the new parameters.
 
 #### Syntax
-`mdns.register(hostname [, hostdesc], servicename, port [, attributes])`
+`mdns.register(hostname [, attributes])`
 
 #### Parameters
 - `hostname` The hostname for this device. Alphanumeric characters are best.
-- `hostdesc` The host description for this device. Short phrase, can include spaces.
-- `servicename` The service name for this device. Alphanumeric characters are best. This will get prefixed with `_` and suffixed with `._tcp`
-- `port` The port number for the primary service.
-- `attributes` A optional table of up to 10 attributes to be exposed. The keys must all be strings.
+- `attributes` A optional table of options. The keys must all be strings.
+
+The `attributes` contains two sorts of attributes -- those with specific names, and all the rest. The specific names are:
+
+- `port` The port number for the service. Default value is 80.
+- `service` The name of the service. Default value is 'http'.
+- `dscription` A short phrase (under 63 characters) describing the service. Default is the hostname.
```

Comment:
`description` not `dscription`

"all the rest": this is very enigmatic.  What and why are they allowed?  Even if we just refer out to a standard or wiki page which defines what optional parameters are allow; this would help a lot.